### ISSUE TITLE
[Feature] Implement S3/HDFS Integrations for CHYT in Ytsaurus

### ISSUE THREAD

**Issue by kaikash** [ytsaurus/ytsaurus#29]
Clickhouse offers valuable integrations like S3, HDFS, and others. Adding support for these integrations in CHYT within Ytsaurus would greatly enhance convenience and functionality.

**Comment by kaikash** [comment]
Any updates here?

**Comment by kaikash** [comment]
Yes, you are correct. We need the implementation of S3 and HDFS table functions in CHYT within Ytsaurus, similar to how it's executed in ClickHouse. Specifically, the feature should allow us to import and export data from and to S3/HDFS, using a syntax akin to this ClickHouse example:

```SQL
INSERT INTO FUNCTION s3('https://clickhouse-public-datasets.s3.amazonaws.com/my-test-bucket-768/test-data.csv.gz', 'CSV', 'name String, value UInt32', 'gzip')
SELECT name, value FROM existing_table;
```

**Comment by kaikash** [comment]
I am not certain if exporting data to HDFS using ClickHouse's [HDFS table function](https://clickhouse.com/docs/en/sql-reference/table-functions/hdfs) is possible.

However, using the [HDFS table engine](https://clickhouse.com/docs/en/engines/table-engines/integrations/hdfs), it can be achieved. Therefore, it might be more preferable to use the HDFS table engine instead of the table functions for exporting data to HDFS.

**Comment by DimasKovas** [comment]
Status update: S3 functions have been supported in CHYT for a few months now. HDFS functions are in the backlog and we do not have plans to support them in the near future.
